### ISSUE TITLE
Bug: Unable to get Gulpfile.js to compile scss

### ISSUE THREAD

**Issue by iris-i** [18F/web-design-standards-drupal-d7#26]
**To Replicate**
1. Download the drupal 7 theme
2. From the theme root, run npm install.
3. Run gulp.

I could not run gulp at all from the top level so I tried running it from the gulp folder. The sass files do not get compiled. 
I created new package.json & gruntfile.js mirroring the existing folder structure, and I put the both files in the top level directory in the theme. This compiles the files. _Although there is a puzzling factor of the main.scss partial residing in the css folder (which belongs in a different issue)_

Shooting in the dark here, but I wonder if the folder structure is different from what it was when the gulpfile was created? Or maybe I missed some glaringly obvious step in getting gulp running.

I guess this is a good time to prioritze #21 
cc: @AlexBrown-appnovation

**Comment by iris-i** [comment]
@AlexBrown-appnovation. Looping the conversation into this issue: 
"The Gulp setup is for custom styles only. You can see what that custom SASS structure is in this PR:

https://github.com/18F/web-design-standards-drupal/pull/19

Because the USWDS is it's own codebase, it's best practice for us to not touch it at all, and merely layer our own styles to override or supplement it. We should never be changing it directly in case a user wants to update it to the latest version released by 18F.

Hope that makes sense!"

**Comment by iris-i** [comment]
Thanks @AlexBrown-appnovation, I'm afraid it is still not quite clear to me. I understand that it is a good practice not to override USWDS styles, but my intention is to port more of the USWDS component styles into SASS files and have it compile into main.scss/all.scss. I am not layering custom styles at this point.

**Comment by iris-i** [comment]
The USWDS styles are already in sass partials and css files, and we are actually not porting the styles over or doing any compiling for those. Talk about reinventing the wheel.

However we do need to get some custom drupal-specific styles into the theme, (eg. fixing the display of the toolbar admin menu) and as such , we still need to get the correct file structure in for further styling to begin. I'm reopening this issue to mark it blocked by the pull request here: https://github.com/18F/web-design-standards-drupal/pull/19

~~Ok, this is a non-issue, but we should probably start working on #21, to prevent further confusion.~~

**Comment by fureigh** [comment]
@iris-i #19 is now merged. Hope this helps.

**Comment by fureigh** [comment]
Hi @iris-i, and happy new year!

You wrote:

> we still need to get the correct file structure in for further styling to begin

Closing #19 appears to have addressed that, so I'm closing this issue. If you see more to be done here, please either reopen this issue (if it has to do with Gulpfile.js compiling SCSS) or open a new issue. Thanks!
